### PR TITLE
[bids] fix file naming

### PR DESCRIPTION
**Pull request by Edouard2laire**
Fix a small bug where the task name was not used in the file naming

### CODE REVIEW

File: +nirs/+bids/Data2BIDS.m [hunk 1]
```diff
@@ -6,7 +6,7 @@ function Data2BIDS(data,folder,participant_id,session,task,run)
     filename=[filename '_ses-' session];
 end
 
-if(nargin<4 || ~isempty(task))
+if(nargin<4 || isempty(task))
     task='func';
```
Comment (AdrianCurtin) [inline +nirs/+bids/Data2BIDS.m:10]: good catch

File: +nirs/+bids/data2JSON.m [hunk 2]
```diff
@@ -5,13 +5,13 @@ function data2JSON(data,filename,task)
...
-fprintf(fid,'\t"SNIRF_createTime": %s\n',snirf.nirs.metaDataTags.SNIRF_createTime);
-fprintf(fid,'\t"filedescription": %s\n',snirf.nirs.metaDataTags.filedescription);
+fprintf(fid,'\t"TaskName": "%s",\n',task);
+fprintf(fid,'\t"Version": %s,\n',snirf.formatVersion);
+fprintf(fid,'\t"MeasurementDate": "%s",\n',snirf.nirs.metaDataTags.MeasurementDate);
+fprintf(fid,'\t"MeasurementTime": "%s",\n',snirf.nirs.metaDataTags.MeasurementTime);
+fprintf(fid,'\t"SNIRF_createDate": "%s",\n',snirf.nirs.metaDataTags.SNIRF_createDate);
+fprintf(fid,'\t"SNIRF_createTime": "%s",\n',snirf.nirs.metaDataTags.SNIRF_createTime);
+fprintf(fid,'\t"filedescription": "%s"\n',snirf.nirs.metaDataTags.filedescription);
 fprintf(fid,'}');
```
Comment (AdrianCurtin) [inline +nirs/+bids/data2JSON.m:15]: This is also fairly insiduous because the json file will not actually read unless these quotes are present, good work

**Review by AdrianCurtin — APPROVED** [review]

File: +nirs/+bids/data2JSON.m (hunk 2 above)
Comment (Edouard2laire) [reply +nirs/+bids/data2JSON.m:15]: thank you :)

**Comment by Edouard2laire** [comment]
Do you know when this PR will be merged ? :)

**Pull request merged**